payjoin-cli/src/main.rs

```diff
@@ -62,7 +62,7 @@ fn cli() -> ArgMatches {
             .num_args(1)
             .help("The password for the bitcoin node"))
         .subcommand_required(true)
-        .arg(Arg::new("ohttp_config")
+        .arg(Arg::new("ohttp_keys")
             .long("ohttp-config")
```

Comment:
is this and the ohttp config in the next line left as is intentionally?

Reply:
I missed it great catch